RecoEgamma/EgammaTools/python/regressionModifier_cfi.py

```diff
@@ -2,28 +2,27 @@
 
 regressionModifier = \
     cms.PSet( modifierName    = cms.string('EGExtraInfoModifierFromDB'),  
-              autoDetectBunchSpacing = cms.bool(True),
-              applyExtraHighEnergyProtection = cms.bool(True),
-              bunchSpacingTag = cms.InputTag("bunchSpacingProducer"),
-              manualBunchSpacing = cms.int32(50),              
-              rhoCollection = cms.InputTag("fixedGridRhoFastjetAll"),
-              vertexCollection = cms.InputTag("offlinePrimaryVertices"),
+
+              rhoCollection = cms.InputTag('fixedGridRhoFastjetAll'),
+              
               electron_config = cms.PSet( # EB, EE
-                                          regressionKey_25ns  = cms.vstring('gedelectron_EBCorrection_25ns', 'gedelectron_EECorrection_25ns'),
-                                          uncertaintyKey_25ns = cms.vstring('gedelectron_EBUncertainty_25ns', 'gedelectron_EEUncertainty_25ns'),
-                                          combinationKey_25ns   = cms.string('gedelectron_p4combination_25ns'),
-                                          
-                                          regressionKey_50ns  = cms.vstring('gedelectron_EBCorrection_50ns', 'gedelectron_EECorrection_50ns'),
-                                          uncertaintyKey_50ns = cms.vstring('gedelectron_EBUncertainty_50ns', 'gedelectron_EEUncertainty_50ns'),
-                                          combinationKey_50ns   = cms.string('gedelectron_p4combination_50ns'),
+                regressionKey_ecalonly  = cms.vstring('electron_eb_ECALonly_lowpt', 'electron_eb_ECALonly', 'electron_ee_ECALonly_lowpt', 'electron_ee_ECALonly'),
+                uncertaintyKey_ecalonly = cms.vstring('electron_eb_ECALonly_lowpt_var', 'electron_eb_ECALonly_var', 'electron_ee_ECALonly_lowpt_var', 'electron_ee_ECALonly_var'),
+                regressionKey_ecaltrk  = cms.vstring('electron_eb_ECALTRK_lowpt', 'electron_eb_ECALTRK', 'electron_ee_ECALTRK_lowpt', 'electron_ee_ECALTRK'),
+                uncertaintyKey_ecaltrk = cms.vstring('electron_eb_ECALTRK_lowpt_var', 'electron_eb_ECALTRK_var', 'electron_ee_ECALTRK_lowpt_var', 'electron_ee_ECALTRK_var'),
```

Comment:
I know, and I don't like any of them. This is not an uncertainty, maybe it is a var, it is not a standard deviation (sigma). It is really an (rather debatable) estimation of the per-particle resolution. I would have preferred to call them all resolution... historical reasons prevent me.